.github/workflows/staging-deployment.yml

```diff
@@ -124,133 +109,56 @@ jobs:
-          # Check if artifacts directory contains the WAR file
-          echo "[INFO] Checking local artifacts..."
-          ls -la artifacts/
-          if [ ! -f "$WAR_FILE" ]; then
-            echo "Error: No WAR file found in artifacts directory"
-            exit 1
-          fi
-          
-          # Check remote directory exists and is writable
-          echo "[INFO] Checking remote directory..."
-          $SSH "if [ ! -d /opt/artemis ]; then echo 'Error: /opt/artemis directory does not exist'; exit 1; fi"
-          $SSH "if [ ! -w /opt/artemis ]; then echo 'Error: /opt/artemis directory is not writable'; exit 1; fi"
-          
-          # Remove old backup if exists
-          echo "[INFO] Remove old artemis.war ..."
-          $SSH "rm -f /opt/artemis/artemis.war.old"
-          
-          # Copy new artemis.war to node
-          echo "[INFO] Copy new artemis.war ..."
-          scp -v -i ~/.ssh/id_rsa "$WAR_FILE" $DEPLOYMENT_USER@$DEPLOYMENT_HOSTS_PRIMARY:/opt/artemis/artemis.war.new
-          if [ $? -ne 0 ]; then
-            echo "Error: Failed to copy WAR file"
-            exit 1
-          fi
-          
-          # Verify the file was copied successfully
-          echo "[INFO] Verify new WAR file..."
-          $SSH '
-            if [ ! -f /opt/artemis/artemis.war.new ]; then
-              echo "Error: No WAR file found at /opt/artemis/artemis.war.new"
-              exit 1
-            fi
-          '
-          
-          # Stop Artemis-Service on node
-          echo "[INFO] Stop artemis.service ..."
-          $SSH sudo systemctl stop artemis
-          
-          # Replace old artemis.war
-          echo "[INFO] Rename old artemis.war ..."
-          $SSH mv /opt/artemis/artemis.war /opt/artemis/artemis.war.old || true
-          echo "[INFO] Rename new artemis.war ..."
-          $SSH mv /opt/artemis/artemis.war.new /opt/artemis/artemis.war
-          
-          # Start Artemis-Service on node
-          echo "[INFO] Start artemis.service ..."
-          $SSH sudo systemctl start artemis
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
 
-      - name: Verify Primary Node Deployment
-        id: verify_deployment
-        timeout-minutes: 10
+      - name: Install Ansible
         run: |
-          while true; do
-            echo "Performing health check..."
-          
-            RESPONSE=$(curl -s -f $HEALTH_CHECK_URL || echo '{"status":"DOWN"}')
-            STATUS=$(echo $RESPONSE | grep -o '"status":"[^"]*"' | cut -d'"' -f4)
-          
-            if [ "$STATUS" = "UP" ]; then
-              echo "Health check passed! Application is UP"
-              exit 0
-            else
-              echo "Health check failed. Status: $STATUS"
-              echo "Waiting 10 seconds before next attempt..."
-              sleep 10
-            fi
-          done
+          python -m pip install --upgrade pip
+          pip install ansible
```

Comment:
`pip3 install -r requirements.txt` in `Install Artemis-Ansible dependencies` already installs ansible.
I don't think this is necessary here.

Reply:
Done